bin/add_attachment.py

```diff
@@ -40,86 +40,118 @@
 import lxml.etree as ET
 import urllib.request
 
-ALLOWED_TYPES = ['pdf', 'pptx', 'zip']
-ATTACHMENT_TYPES = 'Poster Presentation Note Software Supplementary Dataset'.split()
+ALLOWED_TYPES = ["pdf", "pptx", "zip"]
+ATTACHMENT_TYPES = "Poster Presentation Note Software Supplementary Dataset".split()
+
 
 def main(args):
 
-    print(f'Processing attachment for {args.anthology_id}', file=sys.stderr)
+    print(f"Processing attachment for {args.anthology_id}", file=sys.stderr)
 
-    if args.path.startswith('http'):
+    if args.path.startswith("http"):
         _, input_file_path = tempfile.mkstemp()
         try:
-            print('-> Downloading file from {}'.format(args.path), file=sys.stderr)
-            with urllib.request.urlopen(args.path) as url, open(input_file_path, mode='wb') as input_file_fh:
+            print("-> Downloading file from {}".format(args.path), file=sys.stderr)
+            with urllib.request.urlopen(args.path) as url, open(
+                input_file_path, mode="wb"
+            ) as input_file_fh:
                 input_file_fh.write(url.read())
         except ssl.SSLError:
-            print('-> FATAL: An SSL error was encountered in downloading the files.', file=sys.stderr)
+            print(
+                "-> FATAL: An SSL error was encountered in downloading the files.",
+                file=sys.stderr,
+            )
             sys.exit(1)
```

Comment:
What is the rule that triggers the print formatting here? If it is line length, I personally would much prefer

```python
print("-> FATAL: An SSL error was encountered in downloading the files.",
      file=sys.stderr)
```

But I can be convinced to update my personal preferences if there is a good rule.